layouts/partials/article-meta.html

```diff
@@ -1,6 +1,6 @@
 			<p>
 				{{ if not (eq (.Date.Format "2006-01-02") "0001-01-01") }}<time>{{ .Date.Format "2006-01-02" }}</time>{{ end }}
-{{ if default true (index .Site.Params "show-wordcount") }}
+{{ if default true (index .Site.Params "ShowWordCount") }}
```

Comment:
I did a test, I added `{{ jsonify Site.Params }}`. Turns out hugo lowercases all the Params keys before passing it to the theme. This wouldn't have come up for you as all of your keys are single words. So we need to use lowercase, and case in config files can be left up to the user. Also explains why it wasn't working (but I don't know how I missed that it wasn't working before making this PR!)